test/behaviour/connection/transaction/TransactionTest.java

```diff
@@ -32,7 +32,7 @@
         plugin = "pretty",
         glue = "com.vaticle.typedb.client.test.behaviour",
         features = "external/vaticle_typedb_behaviour/connection/transaction.feature",
-        tags = "not @ignore and not @ignore-typedb-client-java and not @ignore-typedb"
```

Comment:
`@ignore-typedb` should only be for things ignored within TypeDB core (and maybe cluster?)

Reply:
Yep - this was probably added to disable a particular failing test a while back, and should since have been removed